### DISCUSSION TITLE
Deploying at sub-path

### DISCUSSION THREAD

**Discussion by af-swilox** [react-static/react-static#1525] [Q&A]
I'm trying set a project up with react-static and currently failing at deploying at a custom subpath. I've got an apache server running and copied the build result of the Typescript template to a folder so it can be reached at http://localhost/test/reactstatic 

I've also configured basePath with "test/reactstatic" as well as set the basepath attribute of the router to "test/reactstatic". At first it looks as if it works. Hoovering the mouse over "blog" or "about" shows the right path, e.g. http://localhost/test/reactstatic/blog but the moment that I click on it, the url in the browser location shows "http://localhost/blog" and the page is empty.

If I enter  http://localhost/test/reactstatic/blog in the browser location manually, the page loads correctly. But the moment I click on the link its again opening the wrong locations without the base path. 

Any ideas what I'm doing wrong?
Thanks!

**Comment by AndrewCreer** [comment]
See https://github.com/react-static/react-static/issues/1474
I am having the same issues.

**Reply by SleeplessByte** [reply]
Yep this. 

As in that issue: this is not a `react-static` issue, but a router issue.

**Reply by AndrewCreer** [reply]
Do you have a feeling about how difficult it would be to use an alternate router?
I would imagine it is non-trivial.
